src/components/ProjectBar/ProjectName.js

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect, useRef, useState } from "react";
 import { useTranslation } from "react-i18next";
 import { useDispatch, useSelector } from "react-redux";
-import { PencilIcon } from "../../Icons";
+import { PencilIcon, TickIcon } from "../../Icons";
```

Comment:
I think we're reaching that critical mass with the Icons component, we've got an issue in the backlog to deal with the existing ones, but maybe going forward we should start factoring them out into their own files? @sra405?

Reply:
Yeah I agree, this file has grown out of all proportion... 😅 Although thankfully I think this is reusing an existing one